server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/model/MessageProperties.java

```diff
@@ -217,7 +217,8 @@ private static boolean outputProperty(MessageProperty p) {
     public enum ReadProfile {
         Metadata(0, FetchGroup.MINIMAL),
         Header(1, FetchGroup.HEADERS),
-        Full(2, FetchGroup.FULL_CONTENT);
+        Fast(2, FetchGroup.HEADERS),
```

Comment:
I don't understand the mapping between ReadProfile and FetchGroup

Reply:
my fault, the FetchGroup was removed by Benoit BTW, I forgot to fetch